variables.tf

```diff
@@ -4,10 +4,17 @@ variable "cluster_name" {
 }
 
 variable "api_servers" {
-  description = "List of URLs used to reach kube-apiserver"
+  description = "List of URLs used by k8s components to reach kube-apiserver"
   type        = "list"
 }
 
+# When not set, the value of var.api_servers will be used.
+variable "api_servers_admin" {
+  description = "List of URLs used by admins to reach kube-apiserver"
```

Comment:
Instead of admins, I think we should say something like

> List of URLs used to reach kube-apiserver from an external network"

Reply:
And above, in line with that,

> List of URLs used to reach kube-apiserver from within the private network of the cluster"

Reply:
Better now @schu?

Reply:
Yes, thanks.